hw/drivers/adc/gpadc_da1469x/src/gpadc_da1469x.c

```diff
@@ -128,22 +132,41 @@ da1469x_gpadc_configure_channel(struct adc_dev *adev, uint8_t cnum,
 {
     uint32_t ctrl;
     uint32_t ctrl2;
+    uint32_t mux;
     uint16_t refmv;
 
     ctrl = GPADC->GP_ADC_CTRL_REG;
     ctrl2 = GPADC->GP_ADC_CTRL2_REG;
+    mux = GPADC->GP_ADC_MUX_REG;
```

Comment:
there is no such register in DA1469x...

Reply:
The definition is from mynewt_runtime/hw/mcu/dialog/mist/include/DA10101AA-ALL.h